feature_selection/item.yaml

```diff
@@ -12,14 +12,14 @@ labels:
   author: orz
 maintainers: []
 marketplaceType: ''
-mlrunVersion: 1.1.0
+mlrunVersion: 1.6.4
 name: feature-selection
-platformVersion: 3.5.0
+platformVersion: 3.6.0
 spec:
   filename: feature_selection.py
   handler: feature_selection
   image: mlrun/mlrun
   kind: job
   requirements: []
 url: ''
-version: 1.4.0
+version: 1.7.0
```

Comment:
just for curiosity why the version jumps from 1.4.0 to 1.7.0? why not 1.5.0?

Reply:
you are right updated to 1.5.0